pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotQueryResource.java

```diff
@@ -398,18 +414,13 @@ private String getQueryURL(String protocol, String hostName, int port) {
     return String.format("%s://%s:%d/query/sql", protocol, hostName, port);
   }
 
-  public String sendPostRaw(String urlStr, String requestStr, Map<String, String> headers) {
+  public void sendPostRaw(String urlStr, String requestStr, Map<String, String> headers, OutputStream outputStream) {
     HttpURLConnection conn = null;
     try {
-      /*if (LOG.isInfoEnabled()){
+      if (LOGGER.isInfoEnabled()) {
```

Comment:
nit: is this really needed?

Reply:
Probably not. In general this longer way to log is less expensive in terms of allocation that using the default:
```
LOGGER.info("some message with {} and {}", param1, param2)
```

Given this default uses varargs, which means that it allocates an array for no reason if INFO is disabled.